### ISSUE TITLE
After exporting a long table to PDF, only a portion of the rows are displayed.

### ISSUE THREAD

**Issue by Oyyko** [XmacsLabs/mogan#897]
![2023-08-06_17-43](https://github.com/XmacsLabs/mogan/assets/66729335/8afa1a95-6616-41bf-a67f-dbfc7c222fda)

As shown in the image, Mogan is attempting to display a long table on a finite-sized page, resulting in subsequent rows being truncated. 

Desired improvement: the ability to automatically paginate the table for display.

**Comment by darcy-shen** [comment]
FYI @tangdouer1005

**Comment by tangdouer1005** [comment]
i try to fix it

**Comment by jingkaimori** [comment]
You can enable page breaking inside a large table, by toggling `Focus->special->table breaking` menu item

**Comment by darcy-shen** [comment]
Is it ok to enable it by default?

**Comment by jingkaimori** [comment]
> Is it ok to enable it by default?

No, in some case table may be corrupted when enabling this option.

https://savannah.gnu.org/bugs/?65003
